src/components/NurseryWithdrawals/PlantingProgressMap.tsx

```diff
@@ -0,0 +1,3 @@
+export default function PlantingProgressList(): JSX.Element {
+  return <p>'I am a map'</p>;
```

Comment:
@charlie-tf to replace this with Map code

Reply:
We can let the map select what data it wants since list versus map views are different. Map needs a single planting site selected for instance.